### ISSUE TITLE
[BUG] creating a project

### ISSUE THREAD

**Issue by baturkacamak** [infinum/eightshift-boilerplate#213]
`npx create-wp-project`

creates an error with path names containing spaces during cloning git

```
$ pwd
/Users/baturkacamak/Local Sites/test-theme/app/public/wp-content/themes/test-theme
$ npx create-wp-project
```

error message as follows
```
 1. Cloning repo
Error - Error: Command failed: git clone https://github.com/infinum/eightshift-boilerplate.git /Users/baturkacamak/Local Sites/test-theme/app/public/wp-content/themes/test-theme
fatal: Too many arguments.
```

I believe you should write the current directory with double quotes in the npx package

**Comment by IvanGrginovInf** [comment]
Should be the same issue as this one https://github.com/infinum/eightshift-frontend-libs/issues/184

**Comment by IvanGrginovInf** [comment]
In short @baturkacamak it fails because of the spaces in the paths, will fix but in the meantime you can try without any spaces in the path:

> /Users/baturkacamak/Local Sites/test-theme/app/public/wp-content/themes/test-theme

**Comment by IvanGrginovInf** [comment]
Fixed in `create-wp-project@2.0.7`
